products/datafusion/api.yaml

```diff
@@ -149,4 +149,32 @@ objects:
         name: 'version'
         output: true
         description: |
-          Current version of the Data Fusion.
\ No newline at end of file
+          Current version of the Data Fusion.
+      - !ruby/object:Api::Type::Boolean
+        name: 'privateInstance'
+        description: |
+          Specifies whether the Data Fusion instance should be private. If set to
+          true, all Data Fusion nodes will have private IP addresses and will not be
+          able to access the public internet.
+        input: true
+      - !ruby/object:Api::Type::NestedObject
+        name: 'networkConfig'
+        description: |
+          Network configuration options. These are required when a private Data Fusion instance is to be created.
+        input: true
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'ipAllocation'
+            description: |
+              The IP range in CIDR notation to use for the managed Data Fusion instance
+              nodes. This range must not overlap with any other ranges used in the Data Fusion instance network.
+            required: true
+            input: true
+          - !ruby/object:Api::Type::String
+            name: 'network'
+            description: |
+              Name of the network in the customer project with which the Tenant Project
```

Comment:
Are the references to `customer project` and `tenant project` important here? It seems strange, but if those are terms that mean something to data fusion then it's probably fine

Reply:
The tenant project part is relevant here, but the "customer" is not. Removed it to avoid confusion.